custom/conf/app.example.ini

```diff
@@ -454,28 +454,31 @@ TRUSTED_FACETS = ; e.g. http://localhost:3000/
 ;; Use comma to separate multiple modes, e.g. "console, file"
 MODE = console
 ;;
-;; Either "Trace", "Debug", "Info", "Warn", "Error", "Critical", default is "Trace"
+;; Either "Trace", "Debug", "Info", "Warn", "Error", "Critical" or "None", default is "Info"
 LEVEL = Info
 ;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Router Logger
 ;;
 ;; Switch off the router log
-;DISABLE_ROUTER_LOG= ; false
+;DISABLE_ROUTER_LOG=false
 ;;
 ;; Set the log "modes" for the router log (if file is set the log file will default to router.log)
 ROUTER = console
 ;;
-;; The level at which the router logs
+;; The log level that the router should log at. (If you are setting the access log, it's recommended to place this at Debug.)
 ;ROUTER_LOG_LEVEL = Info
 ;;
+;; The handler for router logs, it controls the router log format, mainly for debug purpose, two log handlers:
+;; router_logger_v1: default for before 1.16
+;; router_logger_v2: default for 1.16 and later. Developers could set both `LEVEL=Debug` to get the "started" router messages
+;ROUTER_LOG_HANDLER = router_logger_v2
+;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
 ;; Access Logger (Creates log in NCSA common log format)
 ;;
 ;ENABLE_ACCESS_LOG = false
-;; Set the log "modes" for the access log (if file is set the log file will default to access.log)
-;ACCESS = file
```

Comment:
No this was doing something.

Set:

```
ENABLE_ACCESS_LOG=true 
ACCESS=console
```

to see what it does.

Reply:
There was code `Cfg.Section("log").Key("ACCESS").MustString("file")`: https://github.com/go-gitea/gitea/pull/17308/files#diff-dc549231e9faae5a41ba73c252ed91a07a8efbce80db4ba4778be836739abd86L258

Is it a no-op ?

Reply:
Oh, I see the problem .....